hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MetricsPlugin.java

```diff
@@ -76,7 +77,7 @@ public void run(DiagnosticsLogWriter writer) {
         // we set the time explicitly so that for this particular rendering of the probes, all metrics have exactly
         // the same timestamp
         probeRenderer.timeMillis = System.currentTimeMillis();
-        metricsRegistry.render(probeRenderer);
+        probeRenderContext.renderAt(ProbeLevel.DEBUG, probeRenderer);
```

Comment:
This is currently not the case and I do not like to have everything rendered. It should use the configured probe level.

Reply:
Good catch. Put that as a quick fix and forgot to replace it with the config later.